bigdata/src/main/scala/org/w3/banana/bigdata/Bigdata.scala

```diff
@@ -0,0 +1,30 @@
+package org.w3.banana.bigdata
```

Comment:
I sometimes have resolution problem in Intellij IDEA whe I add double package definition. So, I will postpone them for later commits